### ISSUE TITLE
Is it possible to have tests and source in different projects in Visual Studio?

### ISSUE THREAD

**Issue by jamiehumphries** [wallabyjs/public#153]
For normal C# tests, for a project called `WallabyTrial` the tests typically live in a separate project called `WallabyTrial.Tests`.

I am trying to get this to work with JS testing with Wallaby with a folder structure like this:

```
root
|_ WallabyTrial/
  |_ Scripts/
    |_ calc.js
  |_ WallabyTrial.csproj
|_ WallabyTrial.Tests/
  |_ Scripts/
    |_ calc.spec.js
  |_ wallaby.json
  |_ WallabyTrial.Tests.csproj
```

With a `wallaby.json` that looks like this:

``` json
{
  "files": [
    "../WallabyTrial/Scripts/calc.js"
  ],
  "tests": [
    "Scripts/calc.spec.js"
  ],
  "testFramework": "jasmine@2.1.3"
}
```

But the tests in `calc.spec.js` are failing because it does not load `calc.js`.

I had a bit of a play around with putting the `wallaby.json` in the root of the project and referencing it as a solution item or linking to it within the test project but I couldn't make anything stick.

The only thing that I could make work was having another `csproj` file at the root which included the `wallaby.json` at the root and the scripts from both projects. This is a bit unpleasant for obvious reasons and will require adding any new files to the runner project.

I realise that in writing this is all a bit confusing, so I have made an example project and pushed it to [here](https://github.com/jamiehumphries/WallabyTrial), hopefully that will be useful!

Basically the question is whether having tests in separate projects is something that Wallaby might be able to support or if this is something I need to be doing a different way?

**Comment by ArtemGovorov** [comment]
Thanks for the repo, we are working on the solution.

**Comment by ArtemGovorov** [comment]
I have sent you [the pull request](https://github.com/jamiehumphries/WallabyTrial/pull/1).

It's now (since 1.0.8) possible to have wallaby.js config as a solution item.
